domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -414,7 +414,7 @@ private void init() throws Exception {
 
     postgreSQLClientConfig = getPostgreSQLClientConfig(tenantId, schemaName, Envs.allDBConfs());
 
-    if (isEmbedded()) {
+    if (isEmbedded() && postgreSQLClientConfig.containsKey("testing")) {
```

Comment:
Technically it should check for `true` and there is no unit test to cover the new condition.